packages/graphql/src/mutations/marketplace/createListing.js

```diff
@@ -70,6 +70,15 @@ async function createListing(_, input) {
   let tx
   const deposit = contracts.web3.utils.toWei(String(input.deposit), 'ether')
 
+  let version = input.version
+  if (!version) {
+    version = Object.keys(contracts.marketplaces).sort()[0]
```

Comment:
Sorting is by ascendant order so this will use the contract with the oldest version. e.g. '000'
Shouldn't we default to using the most recent contract when creating new listings ?